src/main/components/assessment/assessment-dropInfo-tooltip.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component, ComponentClass } from 'react';
+import { IAssessment } from '../../services/assessment/assessment';
+import { Button } from '../controls/button/button';
+import { LinkIcon } from '../controls/icon/link';
+import { TrashIcon } from '../controls/icon/trash';
+import { I18nContext } from '../i18n/i18n-context';
+import ReactTooltip from 'react-tooltip';
+import { AssessmentRepository } from '../../services/assessment/assessment-repository';
+import { compose } from 'redux';
+import { localized } from '../i18n/localized';
+import { connect } from 'react-redux';
+import { ModelState } from '../store/model-state';
+import { IUMLElement } from '../../services/uml-element/uml-element';
+
+type OwnProps = {
+  assessment: IAssessment | null;
```

Comment:
Is there any difference between this and `assessment?: IAssessment`? Just curious 😊